pkg/istio-agent/agent.go

```diff
@@ -795,12 +796,6 @@ func getKeyCertInner(certPath string) (string, string) {
 
 // newSecretManager creates the SecretManager for workload secrets
 func (a *Agent) newSecretManager() (*cache.SecretManagerClient, error) {
-	// If proxy is using file mounted certs, we do not have to connect to CA.
```

Comment:
Why delete this if a.secOpts.FileMountedCerts {...} and move it to initSdsServer()?

Reply:
The original code looks to be correct (i.e., first CheckWorkloadCertificate(), then check if a.secOpts.FileMountedCerts ). The description of this PR is to fix logs, not to change the workflow. Does the original workflow have bugs? Unless the original workflow has bugs, changing the original workflow does not seem to be necessary.